tslint.json

```diff
@@ -28,7 +28,6 @@
     "no-switch-case-fall-through": true,
     "no-this-assignment": [true, { "allow-destructuring": true }],
     "no-unused-expression": true,
-    "no-unused-variable": true,
```

Comment:
Ok, this is a real problem for the Angular code. Without this, `tslint:fix` doesn't clean up unused imports anymore. 

I don't want to put 1.0 RC on hold for this, but I really don't agree with relaxing our TSLint rules for the `@clr/angular` just because they don't work for our icons. I believe having an extra tslint config in the `icons/` folder overrides the main config.

Reply:
This is actually deprecated by tslint as a rule in general, it wants you to use the tsconfig flags for this instead.